mods/authentication/src/utils/authUtils.ts

```diff
@@ -0,0 +1,61 @@
+import JWT from './jwt'
+import JwtPayload from './jwtPayload'
+import ITokenManager from './ITokenManager'
+
+export declare interface UserToken {
+  accessToken: string
+}
+
+export declare interface TokenResponse {
+  isValid: Boolean
+  data: JwtPayload
+}
+
+export default class AuthUtils {
+  private handler: ITokenManager
+  constructor (handler: ITokenManager) {
+    this.handler = handler
+  }
+  public validateTokenData = (payload: JwtPayload): boolean => {
+    if (
+      !payload ||
+      !payload.iss ||
+      !payload.userId ||
+      !payload.iss ||
+      !payload.role
+    )
+      throw new Error('Invalid Access Token')
+    return true
+  }
+
+  public createTokens = async (
```

Comment:
This method should be unaware of the user. Please, replace `userId` with `accessKeyId`

> Also, consider naming your variables as `accessKeyId`, `role`, and `issue`